pkg/bootstrap/platform/platform.go

```diff
@@ -22,6 +22,15 @@ const (
 	KubernetesServiceHost = "KUBERNETES_SERVICE_HOST"
 )
 
+type PlatformType string
+
+const (
+	PlatformTypeAWS   PlatformType = "aws"
+	PlatformTypeAzure PlatformType = "azure"
+	PlatformTypeGCP   PlatformType = "gcp"
+	PlatformTypeNone  PlatformType = "none"
```

Comment:
`unknown`

Reply:
We can't change to `unknown`. This is documented https://github.com/istio/istio/blob/e2ce47e2cb85306bcf57160ae3c4a9026cb28018/pkg/bootstrap/platform/discovery.go#L31

Reply:
make some sense, i just saw `Unknow` struct there

Reply:
Yes. Thanks @ramaraochavali for the explanation on behalf of me.

@hzxuzhonghu I think that it is a way to change the name of struct, if you mind `Unknown` struct.